web/pgadmin/browser/server_groups/servers/databases/schemas/tables/static/js/table.js

```diff
@@ -121,7 +121,11 @@ define('pgadmin.node.table', [
           applies: ['object', 'context'], callback: 'count_table_rows',
           category: 'Count', priority: 2, label: gettext('Count Rows'),
           enable: true,
-        },
+        },{
```

Comment:
I don't think so we need a separate menu "Generate Table ERD" it should be "Generate ERD" only. It will generate the diagram based on the object.

Reply:
OK

Reply:
Changed to `ERD For Database` and `ERD For Table`